core/templates/dev/head/css/oppia.css

```diff
@@ -1602,6 +1616,121 @@ pre.oppia-pre-wrapped-text {
   word-spacing: 0;
 }
 
+/* Bootstrap overides of buttons */
+.btn {
+  padding: 6px 12px 6px 12px;
+  font-size: 14px;
+  border-radius: 4px;
+}
+.btn-sm,.btn-group-sm > .btn {
```

Comment:
Here and elsewhere:
- Alphabetize property names within each CSS rule.
- Add a space after each comma, when you have multiple selectors for a CSS rule.

Reply:
Moved to correct place and Alphabetized Thanks!